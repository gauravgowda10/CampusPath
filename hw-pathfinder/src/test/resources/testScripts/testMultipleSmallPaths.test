# Find a path with Dijkstra's algorithm

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C
AddNode simpleGraph D
AddNode simpleGraph E
AddNode simpleGraph F
AddNode simpleGraph G
AddNode simpleGraph H
AddNode simpleGraph I
AddNode simpleGraph J
AddNode simpleGraph K
AddNode simpleGraph L
AddNode simpleGraph M
AddNode simpleGraph N

AddEdge simpleGraph A B 10.0
AddEdge simpleGraph A C 1.0
AddEdge simpleGraph C D 1.0
AddEdge simpleGraph D E 1.0
AddEdge simpleGraph E F 1.0
AddEdge simpleGraph F G 1.0
AddEdge simpleGraph G H 1.0
AddEdge simpleGraph H I 1.0
AddEdge simpleGraph I J 1.0
AddEdge simpleGraph J K 1.0
AddEdge simpleGraph K L 1.0
AddEdge simpleGraph L M 1.0
AddEdge simpleGraph M N 1.0
AddEdge simpleGraph N B 1.0

FindPath simpleGraph A B
