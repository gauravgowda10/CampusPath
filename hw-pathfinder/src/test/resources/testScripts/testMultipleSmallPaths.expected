# Find a path with Dijkstra's algorithm

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph
added node D to simpleGraph
added node E to simpleGraph
added node F to simpleGraph
added node G to simpleGraph
added node H to simpleGraph
added node I to simpleGraph
added node J to simpleGraph
added node K to simpleGraph
added node L to simpleGraph
added node M to simpleGraph
added node N to simpleGraph

added edge 10.000 from A to B in simpleGraph
added edge 1.000 from A to C in simpleGraph
added edge 1.000 from C to D in simpleGraph
added edge 1.000 from D to E in simpleGraph
added edge 1.000 from E to F in simpleGraph
added edge 1.000 from F to G in simpleGraph
added edge 1.000 from G to H in simpleGraph
added edge 1.000 from H to I in simpleGraph
added edge 1.000 from I to J in simpleGraph
added edge 1.000 from J to K in simpleGraph
added edge 1.000 from K to L in simpleGraph
added edge 1.000 from L to M in simpleGraph
added edge 1.000 from M to N in simpleGraph
added edge 1.000 from N to B in simpleGraph

path from A to B:
A to B with weight 10.000
total cost: 10.000